### PR TITLE
splitting rows with multiple oclc's in hathi full data and adding to …

### PR DESCRIPTION
**Pull request by banukutlu**
…the final files to be indexed

Issue #219

### CODE REVIEW

**Comment by cdmo** [comment]
It'd be kinda cool to get the wiki page for this updated https://github.com/psu-libraries/psulib_blacklight/wiki/Synthesizing-overlap-data-from-HathiTrust. Or maybe do you think it's not worth keeping up to date? We could just rely on the documentation and code in the rake file. Seems like a completely valid alternative.

**Comment by cdmo** [comment]
probably should rebase too

**Comment by banukutlu** [comment]
> It'd be kinda cool to get the wiki page for this updated https://github.com/psu-libraries/psulib_blacklight/wiki/Synthesizing-overlap-data-from-HathiTrust. Or maybe do you think it's not worth keeping up to date? We could just rely on the documentation and code in the rake file. Seems like a completely valid alternative.

Yes, I will update the wiki, I think it is nice and helpful to have it.

**Comment by banukutlu** [comment]
> > It'd be kinda cool to get the wiki page for this updated https://github.com/psu-libraries/psulib_blacklight/wiki/Synthesizing-overlap-data-from-HathiTrust. Or maybe do you think it's not worth keeping up to date? We could just rely on the documentation and code in the rake file. Seems like a completely valid alternative.
> 
> Yes, I will update the wiki, I think it is nice and helpful to have it.

update wiki as well

File: lib/tasks/hathi_trust.rake [hunk 1]
```diff
@@ -25,9 +30,38 @@ namespace :hathitrust do
...
+      end
+    end
+  end
+
+  desc 'Merge splitted rows to deduped full file'
+  task :merge_split_oclc do
+    print `csvgrep -c 1 -r "," -i hathi_all_dedupe_with_headers.csv > hathi_single_oclc.csv`
 
-    print `cat hathi_field_list.csv hathi_full_dedupe.csv > hathi_full_dedupe_with_headers.csv`
+    print `cat hathi_single_oclc.csv hathi_multi_oclc_split.csv > hathi_full_dedupe_with_headers.csv`
```
Comment (cdmo) [inline lib/tasks/hathi_trust.rake:64]: 👌

**Pull request merged**